### ISSUE TITLE
72B Model PPO Training Time

### ISSUE THREAD

**Issue by KAKSIS** [allenai/open-instruct#484]
Hi, I have a few questions regarding the PPO stage of the 72B model:
﻿
1.How much data was used during the PPO stage of training?

2.According to the official documentation, six nodes were utilized (5 for training and 1 for VLLM inference). Could you provide an estimate of the total training time under this configuration?

**Comment by natolambert** [comment]
Hey! Mentioned in #485. This will be in the next version of the paper for 2):
> The 8B reward model is trains in 9 hours on 8 H100 gpus, while the full 8B RL run takes ~65 hours on 8 GPUs and the full 70B RL run takes ~60 hours on 48 GPUs.
Note, for both of these models we took an earlier than final checkpoint from the run.

For 1) the full training dataset is ~30k prompts. https://huggingface.co/datasets/allenai/RLVR-GSM-MATH-IF-Mixed-Constraints

GSM only can be fewer (8.8k) https://huggingface.co/datasets/allenai/RLVR-GSM
